### DISCUSSION TITLE
How to map <D-c> and <D-v> to copy/paste when attaching to remote nvim instance

### DISCUSSION THREAD

**Discussion by shahamran** [neovide/neovide#2547] [Q&A] [answered]
As suggested in the [FAQ], I mapped Cmd+C and Cmd+V to copy/paste and it works when running neovide locally.

But when attaching to a remote server (with the same config), it doesn't work, which is not surprising, since the remote instance doesn't have `vim.g.neovide` set.

Can anyone help me configure copy-paste correctly?

(local macOS neovide, and remote Linux neovim.)

[FAQ]: https://neovide.dev/faq.html#how-can-i-use-cmd-ccmd-v-to-copy-and-paste

**Comment by shahamran** [comment]
Also, maybe this use-case is another example for the necessity of #1573.

**Reply by fredizzimo** [reply]
Yes, we need to implement https://neovim.io/doc/user/provider.html#ui-paste

**Comment by shahamran** [comment, marked as answer]
Ok, I was wrong about `vim.g.neovide` not being set. The problem was that the configuration for neovide-related options was loaded before neovide was attached. The solution was to configure neovide in a lazier way.

I'm using LazyVim, and moved neovide options from `lua/config/options.lua` to `lua/config/keymaps.lua`.
